hive-metastore/src/test/java/org/apache/iceberg/hive/TestHiveCatalog.java

```diff
@@ -905,12 +916,12 @@ private String defaultUri(Namespace namespace) throws TException {
   public void testUUIDinTableProperties() throws Exception {
     Schema schema = getTestSchema();
     TableIdentifier tableIdentifier = TableIdentifier.of(DB_NAME, "tbl");
-    String location = temp.newFolder("tbl").toString();
+    String location = temp.resolve("tbl").toString();
```

Comment:
same as above and at all the other places, does this produce the same result as before (aka the `tbl` dir is created)?

Reply:
Yes, the tbl directory gets created.